bin/install.sh

```diff
@@ -57,27 +57,20 @@ EOF
     echo && read -p "Would you like to use the development branch? You will get the latest features, but things may break. (y/N)" -n 1 -r -s DEV && echo
     if [ "$DEV" != 'y'  ]; then
       export DOCKER_TAG="production"
+      echo "Screenly OSE version: Production" > ~/OSE_version.md
```

Comment:
Production isn't a version. it's a channel. The version is the hash.

Reply:
Got it, then we need to edit this point in the master too.

Reply:
Yep. If that's from master then we need to fix it there too.